### PR TITLE
Refactor form data service

### PR DESCRIPTION
**Pull request by michaelruocco**

### CODE REVIEW

File: src/main/java/io/digital/patterns/workflow/common/Form.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package io.digital.patterns.workflow.common;
```
Comment (ra-naik) [inline src/main/java/io/digital/patterns/workflow/common/Form.java:1]: If possible, could you please change the package to use home office standard. The source is owned by home office. It should be uk.gov.homeoffice.*.*

Comment (michaelruocco) [reply src/main/java/io/digital/patterns/workflow/common/Form.java:1]: Yeah sure, I was just following the package structure already in place, should this be changed for all the other existing code too, or just this particular class? If it needs applying for the whole repo I'd probably do it in a separate merge request if that's okay with you?

Comment (ra-naik) [reply src/main/java/io/digital/patterns/workflow/common/Form.java:1]: Separate merge is the good option.

**Review by jgar123 — APPROVED** [review]
LGTM

**Pull request merged**